### DISCUSSION TITLE
Request: Image overwrite changes

### DISCUSSION THREAD

**Discussion by andrewfairlie** [craftcms/cms#8801] [Ideas]
### Description
We run a site where editors frequently upload generic filenames like "3.jpg" and misuse the "A file with the name X already exists" modal. In a rush to get their content live, they often press "Replace it" not understanding what they're replacing and then flag to us later that the wrong images are showing on the site.

It would be great if Craft gave us an option to force it to always "Keep both" to keep things safe, and/or show the before/after on the modal, like below to teach the editors what they're about to replace...

![Frame 1](https://user-images.githubusercontent.com/1772888/77155173-0f71a980-6a95-11ea-886b-913367f5aede.png)

**Comment by brandonkelly** [comment]
Good idea!
